### PR TITLE
Analyze using GitLab CI

### PR DESCRIPTION
**Pull request by tomasz-kaminski-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-autotools-gitlab-ci-sc[SonarCloud]!*
+
+It is very easy to analyze a C/C++/Objective-C project with SonarCloud https://sonarcloud.io/documentation/getting-started/gitlab/[on GitLab CI]:
+
+. Create a `sonar-project.properties` file to store your configuration
+. In your `.gitlab-ci.yml` file:
+.. As the `.pre` stage  stage:
+... Download the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[Sonar Scanner]
+... Download the https://docs.sonarqube.org/latest/analysis/languages/cfamily/#header-4[Build Wrapper]
```
Comment (arseniy-sonar) [inline README.adoc:19]: Turns out you can replace all the SQ documentation with (potentially better) SC docs. See https://github.com/sc-sq-c-family-examples/macos-cmake-gh-actions-sc for example

**Review by arseniy-sonar — APPROVED** [review]
Looks good. I think it's best to replace the SQ doc links with SC.

File: README.adoc (hunk 1 above)
Comment (tomasz-kaminski-sonarsource) [reply README.adoc:19]: Thanks updated.

Comment (arseniy-sonar) [reply README.adoc:19]: I don't see the update. Did you update it on gitlab side?
See the links in these steps:
```
Download the Sonar Scanner
Download the Build Wrapper
```
in https://github.com/sc-sq-c-family-examples/windows-msbuild-gh-actions-sc/blob/main/README.adoc

Comment (tomasz-kaminski-sonarsource) [reply README.adoc:19]: Ah, I missed them.

**Pull request merged**